bin/webpack.js

```diff
@@ -1,19 +1,57 @@
 #!/usr/bin/env node
 
+const cp = require("child_process");
+const inquirer = require("inquirer");
+
+function runCommand(command) {
+	cp.exec(command, (error, stdout, stderr) => {
```

Comment:
stdout and stderr should be piped to stdout/err of this process to watch the installation process.

see child_process.spawn options.stdio https://nodejs.org/api/child_process.html#child_process_options_stdio